media-libs/nvidia-vaapi-driver/nvidia-vaapi-driver-0.0.12.ebuild

```diff
@@ -20,3 +20,17 @@ RDEPEND="media-libs/gst-plugins-bad
 DEPEND="${RDEPEND}
 	>=media-libs/nv-codec-headers-11.1.5.1"
 BDEPEND="virtual/pkgconfig"
+
+pkg_postinst() {
+	# Source: https://github.com/elFarto/nvidia-vaapi-driver/blob/master/src/backend-common.c#L13
+	ewarn "If vaapi drivers fail to load, then make sure that you are"
+	ewarn "passing the correct parameters to the kernel."
+	ewarn "nvidia_drm.modeset should be set to 1."
+
+	ewarn "Grub users can append nvidia_drm.modeset=1 to"
```

Comment:
Given the changes to the wiki, I'd rather just point users to the wiki instead of mentioning just grub here. The wiki can be updated much more easily to reflect changes with grub or other boot-loaders.

Reply:
That's a very good point. I was hesitant about giving bootloader-specific instructions.

Reply:
Rather than grub, they can just uncomment the `#options nvidia-drm modeset=1` line in /etc/modprobe.d/nvidia.conf, that's also done by default if nvidia-drivers is built with USE=wayland (so, most desktop profile users that didn't go out of their way to disable it).

Reply:
> Rather than grub, they can just uncomment the `#options nvidia-drm modeset=1` line in /etc/modprobe.d/nvidia.conf, that's also done by default if nvidia-drivers is built with USE=wayland (so, most desktop profile users that didn't go out of their way to disable it).

I was not aware of that feature! Good to know.
However, I am using a desktop profile

Reply:
After some investigation, `wayland` is only enabled by default in the `gnome` and `plasma` "sub-profiles".

The "raw" desktop profile has it disabled: https://github.com/gentoo/gentoo/blob/master/profiles/targets/desktop/make.defaults

Reply:
Ah right, I forgot we didn't flip that on for regular desktop yet.

Reply:
> Ah right, I forgot we didn't flip that on for regular desktop yet.

Regardless of the `nvidia-drivers` ebuild, do you believe it's a bad idea to inform users on a potential fix in case things fail. A bit like the `spotify` ebuild, https://github.com/gentoo/gentoo/blob/master/media-sound/spotify/spotify-1.2.40.ebuild#L127-L129.

Like @kakra, the ebuild could just point to the wiki instead.